python/pyproject.toml

```diff
@@ -27,7 +27,7 @@ runtime_common = [
 ]
 srt = [
     "sglang[runtime_common]", "cuda-python",
-    "sgl-kernel>=0.0.2.post14", "torch", "vllm>=0.6.3.post1,<=0.6.4.post1",
+    "sgl-kernel>=0.0.2.post14", "torch", "vllm==0.6.4.post1",
```

Comment:
As we discussed offline, upgrading torch to 2.5.1 is fine. I've changed vLLM to 0.6.4.post1 because the main branch currently has issues with torch 2.4.0, and we can't upgrade vLLM beyond v0.6.5 due to conflicts with `outlines`. cc @merrymercy